lang/fuzz.go

```diff
@@ -0,0 +1,11 @@
+package lang
```

Comment:
You'll need the standard headers or this will fail the test.

Reply:
like this?